cmd/security-agent/app/app.go

```diff
@@ -124,6 +124,37 @@ func newLogContext() (*config.Endpoints, *client.DestinationsContext, error) {
 	return endpoints, destinationsCtx, nil
 }
 
+func newLogContextCompliance() (*config.Endpoints, *client.DestinationsContext, error) {
+	logsConfigComplianceKeys := config.LogsConfigKeys{
+		UseCompression:          "compliance_config.logs_config.use_compression",
+		CompressionLevel:        "compliance_config.logs_config.compression_level",
+		ConnectionResetInterval: "compliance_config.logs_config.connection_reset_interval",
+		LogsDDURL:               "compliance_config.logs_config.logs_dd_url",
+		LogsNoSSL:               "compliance_config.logs_config.logs_no_ssl",
+		DDURL:                   "compliance_config.logs_config.dd_url",
+		DevModeNoSSL:            "compliance_config.logs_config.dev_mode_no_ssl",
+		AdditionalEndpoints:     "compliance_config.logs_config.additional_endpoints",
+		BatchWait:               "compliance_config.logs_config.batch_wait",
+	}
+	return newLogContext(logsConfigComplianceKeys, "agent-http-intake.logs.")
```

Comment:
should `agent-http-intake.logs.` be part of the config file parameter ? If no, does it make sense to not pass it and just to use the default in the `config` package ?

Reply:
This value will be changed as soon as the dedicated endpoint is available.